src/components/todo.js

```diff
@@ -1,5 +1,10 @@
+import printElements from '../utilities/printTodos'; // eslint-disable-line
```

Comment:
- [ ] Kindly consider avoiding disabling linters because it helps programmers catch errors and potential problems early in the development process when they are easier to fix. This can help to improve the overall quality of the code and reduce the risk of bugs and other problems.